lib/dma.c

```diff
@@ -300,8 +300,8 @@ MOCK_DEFINE(dma_controller_add_region)(dma_controller_t *dma,
 
     assert(dma != NULL);
 
-    snprintf(rstr, sizeof(rstr), "[%p, %p) fd=%d offset=%#lx prot=%#x",
-             dma_addr, (char *)dma_addr + size, fd, offset, prot);
+    snprintf(rstr, sizeof(rstr), "[%p, %p) fd=%d offset=%#llx prot=%#x",
+             dma_addr, dma_addr + size, fd, (unsigned long long)offset, prot);
```

Comment:
how about a helper macro for all these, LLX() or some such?

Reply:
I'm not clear, you want the format string and the arguments replaced by a macro?

Reply:
Let's go with the latter.